docs/src/main/sphinx/functions/lambda.rst

```diff
@@ -15,7 +15,102 @@ Lambda expressions are written with ``->``::
     x -> CAST(x AS JSON)
     x -> x + TRY(1 / 0)
 
+Examples
+--------
+
+Obtain the squared elements of an array column::
+
+    SELECT numbers,
+           transform(numbers, n -> n * n) as squared_numbers
+    FROM (
+        VALUES
+            (ARRAY[1, 2]),
+            (ARRAY[3, 4]),
+            (ARRAY[5, 6, 7])
+    ) AS t(numbers);
+
+.. code-block:: text
+
+      numbers  | squared_numbers
+    -----------+-----------------
+     [1, 2]    | [1, 4]
+     [3, 4]    | [9, 16]
+     [5, 6, 7] | [25, 36, 49]
+    (3 rows)
+
+Besides the values of the array column being manipulated,
+other columns can be captured as well within the lambda expression.
+The following example provides a showcase of this feature
+for calculating the value of ``f(x) = ax + b`` linear function::
+
+    SELECT xvalues,
+           a,
+           b,
+           transform(xvalues, x -> a * x + b) as linear_function_values
+    FROM (
+        VALUES
+            (ARRAY[1,2], 10, 5),
+            (ARRAY[3,4], 4, 2)
+    ) AS t(xvalues, a, b);
+
+.. code-block:: text
+
+     xvalues | a  | b | linear_function_values
+    ---------+----+---+------------------------
+     [1, 2]  | 10 | 5 | [15, 25]
+     [3, 4]  |  4 | 2 | [14, 18]
+    (2 rows)
+
+
+Find the array elements containing at least one value greater than ``100``::
+
+    SELECT numbers
+    FROM (
+        VALUES
+            (ARRAY[1,NULL,3]),
+            (ARRAY[10,20,30]),
+            (ARRAY[100,200,300])
+    ) AS t(numbers)
+    WHERE any_match(numbers, n ->  COALESCE(n,0) > 100);
+    -- [100,200,300]
+
+
+Lambda functions can be also applied in aggregation functions.
+Below is a sample the overly complex calculation of the sum of all elements of a column::
```

Comment:
Dont use "Below" .. use in the following instead.  See https://developers.google.com/style/word-list#letter-b